recipe/recipe.yaml

```
@@ -1,29 +1,31 @@
-{% set name = "coffea" %}
-{% set version = "2025.1.1" %}
-{% set python_min = "3.9" %}
+schema_version: 1
 
+context:
+  name: coffea
+  version: 2025.1.1
+  python_min: 3.9
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: ${{ name|lower }}
+  version: ${{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/coffea-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/coffea-${{ version }}.tar.gz
   sha256: 0744b1c951973951bacb48708e37fe481697f696aa1b89c28db4a9cfc2ce26d6
 
 build:
+  number: 1
   noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: ${{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python 3.9.*
```

```suggestion
    - python ${{ python_min }}.*
```

Fix for bug https://github.com/conda-incubator/conda-recipe-manager/issues/307

---

Just beat me to it by a hair. :)